### DISCUSSION TITLE
How to start a newsletter with the Docusaurus blog?

### DISCUSSION THREAD

**Discussion by RudraSen2** [facebook/docusaurus#7997] [Q&A] [answered]
I have a website made using Docusaurus on which there's a blog and I want to add a newsletter button to it like all other blogs have. I am a beginner and never saw it implemented on any Docusaurus blog. Any Help will be Appreciated.

**Comment by slorber** [comment, marked as answer]
Adding a newsletter subscribe form would be similar to other use-cases like adding a docs rating system of a blog comments system.

See for example https://madrus4u.com/blog/blog-comments

You should use the swizzle CLI and code your subscription form using React. We don't plan to integrate directly Docusaurus with emailing platform APIs.
